demo/admin/src/common/brevoModuleConfig/brevoContactsPageAttributesConfig.tsx

```diff
@@ -110,10 +108,8 @@ export const getBrevoContactConfig = (
                 />
             </>
         ),
-        input2State: (values?: AdditionalFormConfigInputProps) => {
+        input2State: (values: AdditionalFormConfigInputProps) => {
             return {
-                email: values?.email ?? "",
-                redirectionUrl: values?.redirectionUrl ?? "",
                 attributes: {
                     SALUTATION: values?.attributes?.SALUTATION,
                     FIRSTNAME: values?.attributes?.FIRSTNAME,
```

Comment:
Since the values are not nullable anymore, you can remove the '?' here as well

Reply:
If I do so, I get the error:
_Cannot read properties of undefined (reading 'attributes')_, when I open the BrevoContactForm 🤔 

Did we change anything for those values? I think, they still can be undefined

Reply:
In that case the `?` should be readded in the function params in line 111. I do not know why this was removed 🤷🏻‍♀️